wee_slack.py

```diff
@@ -2644,6 +2647,39 @@ def process_message(message_json, eventrouter, store=True, **kwargs):
             channel.store_message(message, team)
         dbg("NORMAL REPLY {}".format(message_json))
 
+    if download:
+        download_files(message_json, **kwargs)
+
+
+def download_files(message_json, **kwargs):
+    team = kwargs["team"]
+    download_location = config.get_string('files_download_location')
```

Comment:
You should use `config.files_download_location` instead.

Note that you have to add `get_files_download_location = get_string` to `PluginConfig` to make that work.